app/models/upvs/message_template.rb

```diff
@@ -82,8 +82,8 @@ def create_message_reply(message, original_message:, author:)
       data: {
         Predmet: message_title
       },
-      recipient_uri: original_message.metadata["sender_uri"],
-      correlation_id: original_message.metadata["correlation_id"],
+      recipient_uri: original_message.metadata&.dig("sender_uri"),
+      correlation_id: original_message.metadata&.dig("correlation_id"),
```

Comment:
tieto & su podla mna smell ze nam chyba null object pattern. metadata by proste mali byt prazdny hash ak su prazdne a nie nil.

Reply:
To sme spravili, aby metadata boli by defautl `{ }`, takze sa mozeme zbavit tychto `&`.